miditoolkit/midi/parser.py

```diff
@@ -589,18 +622,6 @@ def event_compare(event1, event2):
             #                i = i+2
             #                break
 
-            # If there's a note off event and a note on event with the same
```

Comment:
This is useless now as during sorting all `note_off` messages are passed before `note_on` messages at the same tick.